chunkset.c

```diff
@@ -5,12 +5,13 @@
 #include "zbuild.h"
 #include "zutil.h"
 
+// We need sizeof(chunk_t) to be 8, no matter what.
 #if defined(UNALIGNED64_OK)
 typedef uint64_t chunk_t;
 #elif defined(UNALIGNED_OK)
-typedef uint32_t chunk_t[2];
+typedef struct { uint32_t u32[2]; } chunk_t;
```

Comment:
If we really want to support pre-C11 compilers, then the start of the line should be modified as "typedef struct chunk_t {"... so the struct itself has same name as the variable type the typedef declares.

Same below ;)

Reply:
We have stated that we target C99 compilers, so we should make sure we keep it compatible.
I don't see anything really interesting (for us) in C11 anyways.

Reply:
That's why I commented how this particular line should be modified -- here -- instead of continuing the discussion on the original location. Sometimes long conversations start to wander around and it becomes hard to follow which line(s) are being discussed.

Reply:
Done! (Sorry, was away on vacation ;)).